package.json

```diff
@@ -22,12 +22,13 @@
   },
   "main": "node/index.js",
   "dependencies": {
+    "babel-core": "^5",
     "babel-runtime": "^5",
     "lodash": "^3.3.1",
+    "through": "^2.3.6",
     "tiny-sprintf": "^0.3.0"
   },
   "devDependencies": {
-    "babel-core": "^5",
```

Comment:
After further investigation, it looks to be `browserify` related.

Here are some repro steps.

`package.json`
https://gist.github.com/duckpunch/dda52deb32c31c24ed6c

`index.js`
https://gist.github.com/duckpunch/60b93a97b4e49092ec73

1. Put those two in the same directory.
2. `npm install`
3. `browserify index.js -o hi.js`

I get the following.
```
Error: Cannot find module './transforms/es7' from '/path/to/node_modules/jsnetworkx'
    at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
    at load (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /usr/local/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:99:15)
```